assets/js/123validations.js

```diff
@@ -0,0 +1,62 @@
+function content_for_step_is_valid(clicked_button){
+  var step_is_valid = true;
+  var fieldset_id = clicked_button.data('fieldset');
+
+  var step_values = $('#'+fieldset_id).serialize()
+  step_values = serialize_unchecked(fieldset_id, step_values)
+  step_values = step_values.split('&');
+
+  for( var i = 0; i < (step_values.length); i++){
+    var key_value = step_values[i].split('=');
+    if(key_value[0] != "comments"){
+      var field = $('#'+key_value[0]);
+      var value = key_value[1].trim();
+      step_is_valid = (perform_dedicate_validation(field, value) && step_is_valid);
```

Comment:
Line is too long.<br>Identifier 'step_is_valid' is not in camel case.<br>Identifier 'perform_dedicate_validation' is not in camel case.